editor/components/block-settings-menu/plugin-block-settings-menu.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+const { Fill: PluginBlockSettingsMenu, Slot } = createSlotFill( 'PluginsBlockSettingsMenu' );
+
+PluginBlockSettingsMenu.Slot = Slot;
```

Comment:
We should document it somewhere. In general, we should document all Slot/Fill pairs from `editor`. Let's open a follow-up task.

Reply:
yes, I was not sure if we wanted to document it now or wait until we figure out https://github.com/WordPress/gutenberg/pull/7199#discussion_r193858088